include/instantiate.h

```diff
@@ -164,7 +150,7 @@ namespace quda
 
   /**
      @brief This instantiate function is used to instantiate the colors
-     @param[in] U Gauge field
+     @param[in] U Gauge fieldOB
```

Comment:
I think `fieldOB` might be an accident :)

Reply:
e4375a12dfa0144a9c2ece9332a61889c981f964